src/index.js

```diff
@@ -1,20 +1,81 @@
 import './style.css';
 import icon from './icon.png';
+import ToDolist from './todolist.js';
+import { modificarBox, modifyDescription, deleteToDo } from './modify.js';
+// import complete from './delete.js';
 
-const todolist = [{ description: 'Learn webpack', completed: false, index: 1 }, { description: 'Learn React', completed: false, index: 2 }, { description: 'Learn React Native', completed: false, index: 0 }];
+const myToDolist = new ToDolist();
 
-todolist.sort((a, b) => a.index - b.index);
+myToDolist.LoadToDoFromLocal();
 
-// eslint-disable-next-line no-unused-vars
-window.addEventListener('DOMContentLoaded', (event) => {
-  document.querySelector('.todo-list').innerHTML = todolist.map((todo) => `<li class="todo-item">
-                <input class="item-input" type="checkbox" data-index="${todo.index}" id="item${todo.index}" ${todo.completed ? 'checked' : ''}/>
-                <label class="item-label" for="item${todo.index}">${todo.description}</label>
-            </li>`).join('');
+const render = () => {
+  document.querySelector('.ToDoList').innerHTML = '';
+  for (let index = 0; index < myToDolist.ToDo.length; index += 1) {
+    const toDo = myToDolist.ToDo[index];
+
+    const List = document.querySelector('.ToDoList');
+    const element = document.createElement('li');
+    element.classList.add('eachToDo');
+    const completed = document.createElement('input');
+    completed.type = 'checkbox';
+    const inputTask = document.createElement('input');
+    inputTask.type = 'text';
+    inputTask.classList.add('item-input');
+    completed.classList.add('completed');
+    inputTask.value = toDo.description;
+    element.append(completed, inputTask);
+    completed.setAttribute('id', toDo.id);
+    inputTask.disabled = false;
+    completed.checked = toDo.completed;
+    if (completed.checked) {
+      inputTask.style.textDecoration = 'line-through';
+    }
+
+    inputTask.addEventListener('change', () => {
+      modifyDescription(index, inputTask.value, myToDolist);
+    });
+
+    completed.addEventListener('click', () => {
+      modificarBox(index, completed.checked, myToDolist);
+
+      if (completed.checked) {
+        inputTask.style.textDecoration = 'line-through';
+      } else {
+        inputTask.style.textDecoration = 'none';
+      }
+    });
+
```

Comment:
- [ ] As the last code reviewer said 👀: **Kindly create a new file in a modules folder and move your functions there, then import it to your index.js file as demanded by the second project requirement. This is also the best ES6 practice.**